libs/utils/android/workloads/gmaps.py

```diff
@@ -90,20 +91,33 @@ def run(self, out_dir, location_search, swipe_count=5, collect=''):
         # Allow the activity to start
         sleep(1)
 
-        self.tracingStart()
+        script = TargetScript(self._te)
+        self._log.debug('Accumulating commands')
+
+        for i in range(swipe_count):
+            System.hswipe(script, 20, 80, 100, True)
+            script.commands.append('sleep 1')
```

Comment:
My reasoning behind exposing commands was to let the user modify it at will. Perhaps a certain call will generate a lot of unneeded commands and the accumulated commands would need to be filtered out. The user could also very well write
`script.execute('sleep 1')`
instead of 
`script.commands.append('sleep 1')`
but I find it makes a lot more sense to use "append" to a command to the script rather than to use "execute"